plugin.image.dumpert/changelog.txt

```diff
@@ -0,0 +1,19 @@
+v1.0.4 (2019-09-21)
```

Comment:
Please remove the changelog from the submission and use only the `<news>` element in addon.xml

Reply:
ok